ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -424,6 +424,40 @@
 #         divisor_override=None,
 #     )
 
+# def _filter_dtypes(input_dtype):
+#     assume(("bfloat16" not in input_dtype) and ("float16" not in input_dtype))
+
+# # softshrink
+# @handle_frontend_test(
+#     fn_tree="mindspore.ops.function.nn_func.softshrink",
+#     dtype_and_input=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
```

Comment:
We should be testing for all valid dtypes by using the "valid" argument here. And in that case I think there is no need for _filter_dtypes anymore either